.rubocop.yml

```diff
@@ -15,7 +15,7 @@ AllCops:
     - '/Rakefile'
     - '/config.ru'
   Exclude:
-    - 'vendor/*'
+    - 'vendor/**/*'
```

Comment:
This line did the trick!

Reply:
Except that I know have errors in code climate 🌦️

Reply:
oof wow that was obscure! Glad you're figuring it out though!

Reply:
Hmm, i see:

> We had trouble running the rubocop engine.
> The engine's output shown below may indicate the cause of the error.
> If not, please contact us so we can investigate further.

```
.rubocop.yml: `require` is concealed by duplicate
/usr/local/bundle/gems/rubocop-0.70.0/lib/rubocop/config_loader_resolver.rb:196:in `rescue in gem_config_path': Unable to find gem rubocop-shopify; is the gem installed? Gem::MissingSpecError (Gem::LoadError)
	from /usr/local/bundle/gems/rubocop-0.70.0/lib/rubocop/config_loader_resolver.rb:192:in `gem_config_path'
	from /usr/local/bundle/gems/rubocop-0.70.0/lib/rubocop/config_loader_resolver.rb:48:in `block (2 levels) in resolve_inheritance_from_gems'
	from /usr/local/bundle/gems/rubocop-0.70.0/lib/rubocop/config_loader_resolver.rb:46:in `reverse_each'
```

Reply:
Isn't CodeClimate just running rubocop? So maybe we just need one or the other. Rubocop's job ran fine!

> 32 files inspected, no offenses detected

Gosh surprising that this is a lot more complex seeming than our other Rails repos. I guess the setup is relatively different?

Reply:
You're right, I removed the rubocop plugin from codeclimate as it is in GH Actions now.